recipe/build.sh

```diff
@@ -5,22 +5,25 @@ cp $BUILD_PREFIX/share/libtool/build-aux/config.* .
 # need macosx-version-min flags set in cflags and not cppflags
 export CFLAGS="$CFLAGS $CPPFLAGS"
 
+if [[ "${PKG_NAME}" == "libcurl-securetransport-static" ]]; then
+    USESSL="--with-secure-transport"
+    STATIC_BUILD="--enable-static --disable-shared"
+else
+    USESSL="--with-openssl=${PREFIX}"
+fi
```

Comment:
When this script is called, `PKG_NAME` will always be `curl_split_recipe`

Reply:
thanks @isuruf, great catch! sorry